rousseau_vote/lib/src/models/entity.dart

```diff
@@ -14,6 +15,7 @@ class Entity {
   String fullName;
   String slug;
   Profile profile;
+  List<Badge> badges;
```

Comment:
Out of scope, but why "Entity"? Isn't it a too generic name?

Reply:
Entity is the superclass of User, Project and anything else that can be voted.